### PR TITLE
Cleanup unused code

### LINKED ISSUE #11: modDeploy.bas is not relevant to this project
Deployment is a very different project from version control. I would not include these features in the VCS system at all. Another good reason not to include this module is that it contains system specific code that isn't included in the project.

### PR DESCRIPTION
**Pull request by A9G-Data-Droid**
I apologize for the large amount of changes. You can see from  "356 additions and 1,300 deletions. " that I found a lot of code that wasn't needed. 

There is a functional change to the property handler, I cleaned up that strange code used to guess the varType. I also added an explicit type to your version property so it will be reliably set each time.

This is the version I am using internally now. It sets you up to move forward with keeping your source and binaries separate. After this pull you would need to release your binaries manually because they won't be in the code anymore. The version I am using was ran through my AccessDB-Compiler. After this pull you can use the compiler to build your releases. 

Closes #11

### CODE REVIEW

File: Version Control.accda.src/modules/modADPFunctions.bas [hunk 1]
```diff
@@ -76,7 +76,7 @@ Public Function GetSQLObjectModifiedDate(strName As String, ByVal strType As Str
         ' Look up from query, and cache results
         Set colCache = New Collection
         dteCacheDate = Now()
-        strLastType = strType
+        strLastType = strSchemaFilter
```
Comment (A9G-Data-Droid) [inline Version Control.accda.src/modules/modADPFunctions.bas]: strSchemaFilter was declared and never used. It had a clear use that was never implemented until now.

File: Version Control.accda.src/modules/modFunctions.bas [hunk 2]
```diff
@@ -457,14 +457,14 @@ End Sub
...
-Public Function GetSafeFileName(strName As String) As String
+Public Function GetSafeFileName(ByRef strName As String) As String
 
     Dim strSafe As String
 
     ' Use URL encoding for these characters
     ' https://www.w3schools.com/tags/ref_urlencode.asp
     strSafe = Replace(strName, "%", "%25")  ' Since we are using this character for encoding. (Makes decoding easier if we do that at some point in the future.)
-    strSafe = Replace(strName, "<", "%3C")
+    strSafe = Replace(strSafe, "<", "%3C")
```
Comment (A9G-Data-Droid) [inline Version Control.accda.src/modules/modFunctions.bas]: This appears to be a bug where % was not handled as it should.

File: Version Control.accda.src/modules/modVersionControl.bas [hunk 3]
```diff
@@ -145,21 +145,19 @@ Private Sub LoadVersionControl(blnUseVersionControl As Boolean, strLibraryPath A
                     '  load and allows us to continue with calling the VCS functions.)
                     On Error Resume Next
                     References.AddFromFile strPath
-                    If Err Then
```
Comment (A9G-Data-Droid) [inline Version Control.accda.src/modules/modVersionControl.bas]: Redundant IF check, adds nothing

**Comment by joyfullservice** [comment]
Thanks again for these contributions! Several have already been implemented, and some of the others are pending a working version of the RubberDuck add-in. Since this is so far behind in the base code version, I am going to close this pull request for now, and we can open new ones (preferably with manageable sets of changes) to implement into the current code base.

**Pull request closed without merging**